examples/ring_c.c

```diff
@@ -7,8 +7,8 @@
  * Simple ring test program in C.
  */
 
-#include "mpi.h"
```

Comment:
Why is this beautification change part of the sessions PR ?

Reply:
I'll put back the ring_c.c from master.  I think @hjelmn was doing some clang formatting here.

Reply:
No. My guess is that this happened when merging with master. I think the ordering on master changed when the examples were formatted. Anyway, looked like it is fixed now so hopefully there are not any more changes like that. Will spot check and see.